src/NexusMods.App.UI/Controls/Trees/Files/FileTreeNodeView.axaml

```diff
@@ -0,0 +1,55 @@
+<reactiveUi:ReactiveUserControl x:TypeArguments="ui:IViewModelInterface"
+                                xmlns="https://github.com/avaloniaui"
+                                xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                                xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                                xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                                xmlns:reactiveUi="http://reactiveui.net"
+                                xmlns:icons="clr-namespace:Projektanker.Icons.Avalonia;assembly=Projektanker.Icons.Avalonia"
+                                xmlns:ui="clr-namespace:NexusMods.App.UI"
+                                xmlns:files="clr-namespace:NexusMods.App.UI.Controls.Trees.Files"
+                                xmlns:resources="clr-namespace:NexusMods.App.UI.Resources"
+                                xmlns:unifiedIcon="clr-namespace:NexusMods.App.UI.Controls.UnifiedIcon"
+                                mc:Ignorable="d" d:DesignWidth="710" d:DesignHeight="44"
+                                x:Class="NexusMods.App.UI.Controls.Trees.Files.FileTreeNodeView"
+                                d:DataContext="{x:Static files:FileTreeNodeDesignViewModel.SampleFolder}">
+    <Design.DataContext>
+        <!-- This ViewModel has a bool switch 'showMultipleRoots' to toggle a view which also has Saves -->
+        <files:FileTreeNodeDesignViewModel />
+    </Design.DataContext>
+    <Grid ColumnDefinitions="*, Auto" Margin="0,5,0,5">
+        <Grid.Styles>
+            <Style Selector="unifiedIcon|UnifiedIcon.FileTypeIcon">
+                <Setter Property="Size" Value="18" />
+                <Setter Property="Margin" Value="8, 0, 0, 0" />
+            </Style>
+
+            <Style Selector="unifiedIcon|UnifiedIcon.View">
+                <Setter Property="Size" Value="20" />
+            </Style>
+
+            <Style Selector="Button.RightElements">
+                <Setter Property="Margin" Value="0,0,8,0" />
+            </Style>
+
+            <Style Selector="TextBlock#FileNameTextBlock">
+                <Setter Property="Margin" Value="4,0,8,0" />
+            </Style>
+        </Grid.Styles>
+
+        <!-- Left Elements -->
+        <Grid Grid.Column="0" ClipToBounds="True" ColumnDefinitions="Auto,Auto" Name="FileElementGrid">
+            
+            <!-- File / Directory Icon -->
+            <unifiedIcon:UnifiedIcon Grid.Column="0" Classes="File FileTypeIcon" IsVisible="False" x:Name="FileEntryIcon" />
+            <unifiedIcon:UnifiedIcon Grid.Column="0" Classes="FolderOutline FileTypeIcon" IsVisible="False" x:Name="FolderEntryIcon" />
+            <unifiedIcon:UnifiedIcon Grid.Column="0" Classes="FolderOpenOutline FileTypeIcon" IsVisible="False" x:Name="FolderOpenEntryIcon" />
```

Comment:
A single `UnifiedIcon` can be used. Instead of switching `IsVisible`, you add/remove the appropriate class.

Reply:
This code was adjusted to `UnifiedIcon` late into dev, previously it used regular icons and same design/approach as 'AdvancedInstaller'.

Should the AdvancedInstaller code also therefore be amended as part of this then?